content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -0,0 +1,164 @@
+draft: false
+images: []
+menu:
+  developers:
+    parent: "developer-patterns"
+weight: 112
+toc: true
+aliases: 
+  - /docs/developer-patterns/scitt-api/
+---
+
+{{< caution >}}
+The SCITT API is currently in preview and subject to change
+{{< /caution >}}
+
+The **S**upply **C**hain **I**ntegrity, **T**ransparency and **T**rust (SCITT) initiative is a set of [IETF standards](https://datatracker.ietf.org/group/scitt/documents/) for managing the compliance and auditability of goods and services across end-to-end supply chains.
+SCITT supports the ongoing verification of goods and services where the authenticity of entities, evidence, policy, and artifacts can be assured and the actions of entities can be guaranteed to be authorized, non-repudiable, immutable, and auditable.
+
+To assure insights to supply chain artifacts are current, the SCITT APIs provide a correlation of statements for a specific artifact, allowing newer information to be registered for the most up to date insights.
+
+This quickstart will:
+
+1. create, or use an existing a key to sign a collection of statements about an artifact
+1. create and register an SBOM for the artifact
+1. create and register an attestation for the artifact
+1. query a collection of statements about the artifact
+
+## Prerequisites
+
+- [A DataTrails subscription](https://app.datatrails.ai/signup)
+- [DataTrails sample code](#datatrails-sample-code)
+- [SBOM Tool](https://github.com/microsoft/sbom-tool)
+
+### DataTrails Sample Code
+
+Clone the [DataTrails SCITT Examples](https://github.com/datatrails/datatrails-scitt-samples) repository
+
+```bash
+git clone https://github.com/datatrails/datatrails-scitt-samples.git
+
+cd datatrails-scitt-samples
+```
+
+## Create a Signing Key
+
+{{< note >}}
+If you already have a COSE Key, skip ahead to [Register a SBOM for the Artifact](#register-a-sbom-for-the-artifact)
+{{< /note >}}
+
+DataTrails supports multiple signing keys.
+_(TODO: link to supported DataTrails Signing Keys\)_<br>
+For a quickstart, we'll create a testing [COSE Key](https://cose-wg.github.io/cose-spec/#key-structure) which DataTrails will cryptographically validate upon registration
+
+1. Create a local signing key
+
+    ```shell
+    openssl ecparam -name prime256v1 -genkey -out scitt-signing-key.pem
+    ```
+
+## Register a SBOM for the Artifact
+
+When registering statements about an artifact, a common identifier is required to correlate a collection of statements.
+In SCITT, a `CWT_Claims subject` property captures the identifier.
+
+The example generates an [SPDX SBOM](https://spdx.dev/), which generates a [Document Namespace](https://spdx.github.io/spdx-spec/v2.2.2/document-creation-information/#65-spdx-document-namespace-field) field which is used for the `CWT_Claims subject` property.
+
+_\<TODO: Add a doc for creating unique identifiers>_
+
+1. Using the [SBOM Tool](https://github.com/microsoft/sbom-tool), generate an SPDX SBOM for `artifact.js`
```

Comment:
minor: just noticed all of these steps are labelled 1?

Reply:
yeah, markdown formatting numbers them sequentially when rendered. This way inserting a step in the middle doesn't require renumbering everything.